com.unity.multiplayer.mlapi/Runtime/Messaging/MessageBatcher.cs

```diff
@@ -29,13 +30,23 @@ public SendStream()
         // Stores the stream of batched RPC to send to each client, by ClientId
         private readonly Dictionary<ulong, SendStream> m_SendDict = new Dictionary<ulong, SendStream>();
 
+        public void Shutdown()
```

Comment:
Could this implement `IDisposable` and use `Shutdown` as the `Dispose` function maybe?

Reply:
Sure, it could. The reason I didn't was to be consistent with other things - NetworkManager, NetworkTransport, MessageQueueContainer, MessageQueueProcessor, etc all have ShutDown functions, and this one is called from MessageQueueProcessor.Shutdown() and NetworkManager.Shutdown() (each of which have their own message batcher for some reason), so it seemed appropriate to follow the same naming.

Reply:
if we're relying on GC to clean up after ourselves because we just forgot to, we're already ruined :P